pkg/acsengine/engine.go

```diff
@@ -873,6 +872,20 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 			}
 			return strings.TrimSuffix(buf.String(), ", ")
 		},
+		"GetAPIServerConfigKeyVals": func(kc *api.KubernetesConfig) string {
+			apiServerConfig := kc.APIServerConfig
+			// Order by key for consistency
+			keys := []string{}
+			for key := range apiServerConfig {
+				keys = append(keys, key)
+			}
+			sort.Strings(keys)
+			var buf bytes.Buffer
+			for _, key := range keys {
+				buf.WriteString(fmt.Sprintf("\\\"%s=%s\\\", ", key, apiServerConfig[key]))
```

Comment:
The escaped slashes `\\` in front of the escaped quote `\"` is needed in order to produce a literal quote char in the resulting yaml file.

This should be done in the above getters as well, will do so in a follow-up PR.